src/lib.rs

```diff
@@ -421,9 +421,6 @@
 #![warn(unreachable_pub)]
 #![deny(dead_code)]
 #![cfg_attr(not(any(feature = "std", test)), no_std)]
-// can remove this if/when rustc-serialize support is removed
```

Comment:
Don't think this was intended?

Reply:
yeah not sure what happened there! I've fixed and re-squashed